docs/backend-api/resources/tracking/geocoder.md

```diff
@@ -55,26 +53,52 @@ Performs a forward geocoding. Returns a list of locations matching the given add
             ...
         ]
     }
+```
 
+where **details_object** is:
+
+```javascript
+{
+    "country": <string>, // optional
+    "province": <string>, // optional
+    "locality": <string>, // optional
+    "street": <string>, // optional
+    "house": <string>, // optional
+    "postcode": <string> // optional
+}
+```
 ---
-## search_location(…)
+## Search location
+
+### search_location(…)
+
+Search address by location using geocoder
+
+#### structure:
+
+    [api_base_url]/geocoder/search_location?hash=your_hash&location={"lat": , "lng": }
 
 #### parameters:
 
-*   location – not null, **location** (see: [data types description section](../../getting-started.md#data-types) section)
-*   geocoder – optional, string (enum). Now supported `google`, `osm`, `yandex`, `locationiq` geocoders.
-*   lang – optional, string, ISO 639 language code
-*   with_details – optional, boolean. If true then the response will contain details
-*   goal - optional, string (enum). Now supported `ui`, `ui_user_action`. Helps to choose the target geocoder. 
-Use `ui_user_action` for requests initiated by user, otherwise `ui`.
+| name | description | type | format |
+| :------: | :------: | :-----:| :-----:|
+| location | location coordinates (see: [data types description section](../../getting-started.md#data-types) section) | location | {"lat": , "lng": } |
+| geocoder | optional, geocoder type that will be used for searching | string (enum) | google |
+| lang | optional. ISO 639 [language code](../../getting-started.md#data-types) | locale | en_US |
+| with_details | optional. If true then the response will contain details | boolean | true/false |
+| goal | Helps to choose the target geocoder. Now supported `ui`, `ui_user_action`. Use `ui_user_action` for requests initiated by user, otherwise `ui` | string (enum) | ui | 
 
-Search address by location.
-User language wil be used If `lang` is omitted.
+#### example:
+
+    [api_base_url]/geocoder/search_location?hash=your_hash&location={"lat": 56.827001, "lng": 60.594296}

```

Comment:
This is wrong request. Better to use `cURL` format with `Content-Type: application/json`.